server/server.js

```diff
@@ -271,9 +318,14 @@ io.on('connection', (socket) => {
                     lobby.correctGuesses >
                     numberOfPlayers - 1 - lobby.correctGuesses
                 ) {
+                    lobby.pointMap.set(
```

Comment:
A `lobby.pointMap.get(lobby.users[lobby.drawerSocketId])` lekérdezés értelmezhetetlenül hosszú. Itt jó lenne egy komment, hogy mit csinál ez a három sor vagy kivinni részeit változókba pl `drawerPrevPoints` vagy `drawerUserName`

Reply:
és a többi esetben is.